server/sonar-web/src/main/js/apps/overview/qualityGate/QualityGateCondition.js

```diff
@@ -124,8 +123,7 @@ export default class QualityGateCondition extends React.PureComponent {
           className={className}
           component={component.key}
           metric={condition.measure.metric.key}
-          period={condition.period}
-          periodDate={periodDate}>
+          sinceLeakPeriod={true}>
```

Comment:
`true` must not be passed always, but only if `condition.period` is defined

For example, add a quality gate condition `issues > 0` and analyze a project with issues. The link will be wrong pointing only to the leak period.